tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -78,9 +78,9 @@ $ sudo pip3 install --upgrade https://storage.googleapis.com/tensorflow/mac/tens
 ```
 
 
-You can now [test your installation](#test_install).
+You can now [test your installation](#test-the-tensorflow-installation).
 
-## Virtualenv installation {#virtualenv_install}
```

Comment:
That would be cool -- basically, we can get rid of all {#...} tags. We just
have to make sure we find all the occurrences. If your code can do that,
awesome. Otherwise, it's not a particularly hard perl script to write.

I have to do some plumbing to make sure I can test the website with those
changes, but if you send a PR, I will do that.

On Wed, Dec 30, 2015 at 12:45 PM Sam Abrahams <notifications@github.com>
wrote:

> In tensorflow/g3doc/get_started/os_setup.md
> <https://github.com/tensorflow/tensorflow/pull/648#discussion_r48630264>:
>
> >
> > -## Virtualenv installation {#virtualenv_install}
>
> Good to know. If we want to make more of these changes to the rest of the
> non-auto-generated Markdown, I can modify the script I used in #584
> <https://github.com/tensorflow/tensorflow/pull/584> to do so.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/648/files#r48630264>.
>